os/include/tinyara/net/if/wifi.h

```diff
@@ -541,6 +559,8 @@ typedef trwifi_result_e (*trwifi_get_info)(struct netdev *dev, trwifi_info *info
  */
 typedef trwifi_result_e (*trwifi_drv_ioctl)(struct netdev *dev, trwifi_msg_s *msg);
 
+typedef trwifi_result_e (*trwifi_get_deauth_reason)(struct netdev *dev, int *deauth_reason);
```

Comment:
```suggestion
typedef trwifi_result_e (*trwifi_get_deauth_reason)(struct netdev *dev, int *deauth_reason);
```
Could you remove this line?